File: 2248-minimum-cost-of-buying-candies-with-discount/minimum-cost-of-buying-candies-with-discount.java
class Solution {
    public int minimumCost(int[] cost) {
        Arrays.sort(cost);
        int t=0,m=0;
        for(int i=cost.length-1;i>=0;i--){
            if(m%3!=2){
                t+=cost[i];
            }
            m++;
        }
        return t;
    }
}

File: 2248-minimum-cost-of-buying-candies-with-discount/minimum-cost-of-buying-candies-with-discount.cs
public class Solution {
    public int MinimumCost(int[] cost) {
        Array.Sort(cost);
        Array.Reverse(cost);
        int t=0;
        for(int i=0; i<cost.Length; i++){
            if(i%3!=2){
                t+=cost[i];
            }
        }
        return t;
    }
}